recipes/default.rb

```diff
@@ -241,6 +270,13 @@
   action :delete
 end
 
+# Another Hortonworks mess to clean up
```

Comment:
lol... a second comment explaining that HDP packages copy default configs here would be helpful